vault.py

```diff
@@ -57,12 +57,17 @@ def run(self, terms, inject=None, variables=None, **kwargs):
         if not url:
             raise AnsibleError('VAULT_ADDR environment variable is missing')
 
-        token = os.getenv('VAULT_TOKEN')
+        # the environment variable overrides file-based token
+        token = os.getenv('VAULT_TOKEN') or open(os.getenv("HOME") + "/.vault-token").read()
```

Comment:
This will throw and `IOError` if the file does not exist, I would silently fail and extend the error message in the later check for token. Make sure that the opened file get closed afterwards, use `with:` perhaps